frontend/playwright.config.js

```
@@ -84,13 +93,32 @@ export default defineConfig({
+    // webServer under vedvarende belastning er et kendt community-mønster
+    // (vitejs/vite#12883, microsoft/playwright#21227); at skifte til en dedikeret
+    // static-file-server er den dokumenterede workaround. sirv er samme
+    // minimale, afprøvede library svelte-værktøjerne bruger — ingen af vites
+    // egen preview-serverstack. Kaldes via `npm run` (ikke `npx`) for at holde
+    // præcis ÉT dræbeligt barn i process-træet, samme mønster som virker i dag.
+    // NB: sirv har ingen middleware-hook som vites `configurePreviewServer`, så
+    // worktreeIdPlugin (vite.config.js) servede ikke længere WORKTREE_ID_PATH —
+    // "preview:e2e" (package.json) genererer den nu som en statisk fil i dist/
+    // via scripts/write-worktree-id.mjs FØR sirv starter. Uden den fejlede
+    // false-green-guarden (global-setup.js) højlydt på sirv selv.
     command: process.env.CI
```

ESLint

[error] 'process' is not defined.

(no-undef)